### PR TITLE
Move to new github.com/sirupsen/logrus.

### PR DESCRIPTION
**Pull request by rhatdan**
Need to mv to latest released and supported version of logrus
switch github.com/Sirupsen/logrus github.com/sirupsen/logrus

Also vendor in latest containers/storage and containers/image

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
Replaces https://github.com/kubernetes-incubator/cri-o/pull/716

File: vendor.conf [hunk 1]
```diff
@@ -63,9 +63,9 @@ github.com/coreos/pkg v3
 github.com/golang/groupcache b710c8433bd175204919eb38776e944233235d03
 github.com/fsnotify/fsnotify 7d7316ed6e1ed2de075aab8dfc76de5d158d66e1
 github.com/emicklei/go-restful 09691a3b6378b740595c1002f40c34dd5f218a22
-github.com/Azure/go-ansiterm fa152c58bc15761d0200cb75fe958b89a9d4888e
-github.com/Microsoft/go-winio v0.3.7
-github.com/Microsoft/hcsshim v0.5.11
+github.com/Azure/go-ansiterm master
```
Comment (mrunalp) [inline vendor.conf]: We should use specific commits instead of master so it is easier to debug if there are issues.

Comment (runcom) [inline vendor.conf]: +1

**Comment by rhatdan** [comment]
Had to add  pkg/registrar to make this work, since this was removed from upstream docker.  Will need to look into how we can remove this package going forward.

**Comment by rhatdan** [comment]
@mrunalp @runcom PTAL.  E2E passes.  Just waiting for travis...

**Comment by mrunalp** [comment]
Looks fine. Will wait for travis.

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**